File: exercises/small_problems_101/easy_2/tip_calculator.rb
puts "What is the bill?"
bill = gets.chomp.to_f

puts "What is the tip percentage?"
tip_percent = gets.chomp.to_f

tip = (bill * tip_percent / 100).round(2)
total = (bill + tip).round(2)

puts "The tip is $#{format("%.2f", tip)}"
puts "The total is $#{format("%.2f", total)}"

File: frontend/javascript/exercises/easy_1/tip_calculator.js
var bill = parseFloat(prompt('What is the bill?'));
var percentage = parseFloat(prompt('What is the tip percentage')) / 100;
var tip = bill * percentage;
var total = bill + tip;
console.log('The tip is $' + tip.toFixed(1));
console.log('The total is $' + total.toFixed(1));
